js/mathOperations.js

```diff
@@ -77,28 +80,76 @@ function create()
   myscore = game.add.text(80, 43 , '000' , {font : "15px Arial" , fill : "#eceff1"});
   mylevel = game.add.text(311, 43 , '01' , {font : "15px Arial" , fill : "#eceff1"});
   timer = game.add.text(515, 43, '00:00:00' ,{font : "15px Arial" , fill : "#eceff1"});
-
-  plus = game.add.sprite(150 , 350 , 'plus');
-  minus = game.add.sprite(250 , 350 , 'minus');
-  multiply = game.add.sprite(350 , 350 , 'multiply');
-  divide = game.add.sprite(450 , 350 , 'divide');
-
+  //if(level < 2)
+  //{
+   //plus = game.add.sprite(150 , 350 , 'plus');
+   //minus = game.add.sprite(250 , 350 , 'minus');
+   //multiply = game.add.sprite(350 , 350 , 'multiply');
+   //divide = game.add.sprite(450 , 350 , 'divide');
+   //power = game.add.sprite( 1000 , 1000 , 'power')
+  //} 
+  //else
+  //{ 
+  plus = game.add.sprite(50 , 350 , 'plus');
+  minus = game.add.sprite(150 , 350 , 'minus');
+  multiply = game.add.sprite(250 , 350 , 'multiply');
+  divide = game.add.sprite(350 , 350 , 'divide');
+  power = game.add.sprite(450 , 350 , 'power');
+  //}
   plus.inputEnabled = true;
   minus.inputEnabled = true;
   multiply.inputEnabled = true;
   divide.inputEnabled = true;
-
+  power.inputEnabled = true;
   
  minuskey = game.input.keyboard.addKey(Phaser.Keyboard.S);
  pluskey =  game.input.keyboard.addKey(Phaser.Keyboard.A);
-  dividekey = game.input.keyboard.addKey(Phaser.Keyboard.D);
-  multiplykey = game.input.keyboard.addKey(Phaser.Keyboard.M);
-  pausekey = game.input.keyboard.addKey(Phaser.Keyboard.P);
+ dividekey = game.input.keyboard.addKey(Phaser.Keyboard.D);
+ multiplykey = game.input.keyboard.addKey(Phaser.Keyboard.M);
+ powerkey = game.input.keyboard.addKey(Phaser.Keyboard.F);
+ pausekey = game.input.keyboard.addKey(Phaser.Keyboard.P);
 
   startScreen=game.add.sprite(0,0,'start_screen');
   startButton=game.add.sprite(560,465,'start_button');
   startButton.inputEnabled = true;
   startButton.events.onInputUp.add(startingGame);
+   
+ /*  pluskey.onDown.add(answeredPlus,this);
```

Comment:
Whenever I remove those lines, I am getting an errors(which i hadn't solved). For checking purpose I hadn't removed those lines. I am still investing why i am getting those errors?